src/Hazelcast.Net.Tests/Clustering/MemberPartitionGroupServerTests.cs

```diff
@@ -98,6 +103,47 @@ await AssertEx.SucceedsEventually(()
             AssertClientOnlySees(client3, address3);
         }
 
+        [TestCase(RoutingModes.MultiMember)]
+        [TestCase(RoutingModes.SingleMember)]
+        [TestCase(RoutingModes.AllMembers)]
+        public async Task TestClientCollectsClusterEvents(RoutingModes routingMode)
+        {
+            var address1 = "127.0.0.1:5701";
+            var address2 = "127.0.0.1:5702";
+            var keyCount = 3 * 271;
+
+            // create a client with the given routing strategy for catching the events.
+            var client1 = await CreateClient(RoutingStrategy.PartitionGroups, address1, routingMode);
+            var client1Count = 0;
+            var mapName = $"map_{routingMode}";
+            var map1 = await client1.GetMapAsync<int, int>(mapName);
+
+            await map1.SubscribeAsync((eventHandler) =>
+            {
+                eventHandler.EntryAdded((sender, args) =>
+                {
+                    Interlocked.Increment(ref client1Count);
+                });
+            }, false);
+
+
+            // create a client with the given routing strategy for creating events
+            var client2 = await CreateClient(RoutingStrategy.PartitionGroups, address2, RoutingModes.SingleMember);
```

Comment:
comment says "with the given routing strategy", but we explicitly providing `RoutingModes.SingleMember`

Reply:
Right, fixed at https://github.com/hazelcast/hazelcast-csharp-client/pull/907/commits/7a6a44c4fe4d14654f458efd48f31189cbde1e19